plugins/wazo-gigaset/N510/templates/base.tpl

```diff
@@ -44,8 +44,8 @@
     <SYMB_ITEM ID="BS_IP_Data1.aucS_STUN_SERVER{{ line_suffix }}" class="symb_item" value='""'/>
     <SYMB_ITEM ID="BS_IP_Data1.aucS_OUTBOUND_PROXY{{ line_suffix }}" class="symb_item" value='""'/>
     <SYMB_ITEM ID="BS_IP_Data1.aucS_SIP_PROVIDER_NAME{{ line_suffix }}" class="symb_item" value='"WazoPBX"'/>
-    <SYMB_ITEM ID="BS_IP_Data1.uiI_SIP_SERVER_PORT{{ line_suffix }}" class="symb_item" value="0x{{ "%x"|format(sip_proxy_port) }}"/>
-    <SYMB_ITEM ID="BS_IP_Data1.uiI_SIP_REGISTRAR_PORT{{ line_suffix }}" class="symb_item" value="0x{{ "%x"|format(sip_proxy_port) }}"/>
+    <SYMB_ITEM ID="BS_IP_Data1.uiI_SIP_SERVER_PORT{{ line_suffix }}" class="symb_item" value="0x{{ "13C4"|format(sip_proxy_port) }}"/>
+    <SYMB_ITEM ID="BS_IP_Data1.uiI_SIP_REGISTRAR_PORT{{ line_suffix }}" class="symb_item" value="0x{{ "13C4"|format(sip_proxy_port) }}"/>
```

Comment:
This is not how `|format` works. If what you want is a default value, you have to stack another filter on top: 
https://jinja.palletsprojects.com/en/2.11.x/templates/#default
That would look like `"%x"|format(sip_proxy_port)|default(5060)`. The `"%x"` is mandatory in that case, since Gigaset wants that port number in hexadecimal.